examples/express-composition/src/__tests__/acceptance/note.acceptance.ts

```diff
@@ -57,4 +58,15 @@ describe('NoteApplication', () => {
   async function givenNoteRepository() {
     noteRepo = await lbApp.getRepository(NoteRepository);
   }
+
+  async function changeDataSourceConfig() {
+    /**
+     * Override default config for DataSource for testing so we don't write
+     * test data to file when using the memory connector.
```

Comment:
Have you considered using the approach we are recommending in https://loopback.io/doc/en/lb4/Testing-your-application.html#create-a-test-datasource? Maybe it's not a good one, IDK.

It would be great to have a single prescribed way, used consistently both in our docs and the example repos. I think that's out of scope of this pull request though.

Reply:
I was following how it was done in the `Todo` example, but I can definitely look into that way and update the examples/the docs in a different PR.